### PR TITLE
fix: solve the problem of overlapping elements

### PR DESCRIPTION
**Pull request by cieldon32**
### 🤔 This is a ...

- [ ] Bug fix
### 💡 Background and solution
inline的表单，formItem是input的时候， 且用Row布局，label和input 元素重叠
解决办法是，加样式： flex: 1 0 auto

 https://codesandbox.io/s/zidingyibiaodankongjian-ant-design-demo-50k7n?file=/index.js

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

### CODE REVIEW

**Comment by cieldon32** [comment]
demo： https://codesandbox.io/s/zidingyibiaodankongjian-ant-design-demo-50k7n?file=/index.js

File: components/form/style/index.less [hunk 1]
```diff
@@ -71,6 +71,7 @@
   // ==============================================================
   &-label {
     display: inline-block;
+    flex: 1 0 auto;
```
Comment (zombieJ) [inline components/form/style/index.less:74]: 下面设置了 `flex-grow: 0`不希望缩放，这边直接简化成 `flex: none` 就行了。

File: components/form/style/index.less [hunk 2]
```diff
@@ -71,6 +71,7 @@
   // ==============================================================
   &-label {
     display: inline-block;
+    flex: none;
     flex-grow: 0;
```
Comment (yoyo837) [inline components/form/style/index.less:75]: 有了 `flex: none;` `flex-grow: 0;`是不是可以删了？

Comment (afc163) [reply components/form/style/index.less:75]: @cieldon32

**Comment by zombieJ** [comment]
\rebase

**Pull request merged**